template/haskell-http/.ghc.environment.x86_64-linux-8.6.5

```diff
@@ -0,0 +1,136 @@
+-- This is a GHC environment file written by cabal. This means you can
```

Comment:
This file shouldn't be here.. please [remove it from version control](https://superuser.com/questions/1397199/how-to-ignore-a-tracked-file-in-git-without-deleting-it)